internal/worker/credentialvalidator/worker.go

```diff
@@ -76,6 +76,7 @@ func NewWorker(ctx context.Context, config Config) (worker.Worker, error) {
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+	config.Logger.Infof(ctx, "WatchModelCredential success for mcw")
```

Comment:
delete me

Reply:
Hi ian, sorry was in the midst of QA testing, and needed to push this change for git pulling onto my vm, will revert this thanks!